runtime/src/serde_snapshot.rs

```diff
@@ -418,8 +429,22 @@ where
         snapshot_version,
         snapshot_slot,
         snapshot_bank_hash_info,
+        _snapshot_prior_roots,
+        _snapshot_prior_roots_with_hash,
     ) = snapshot_accounts_db_fields.collapse_into()?;
 
+    /*
+    todo in future pr:
+    deserialize prior_roots and prior_roots_with_hash
+    example:
+    {
+        let mut writer = accounts_db.accounts_index.roots_tracker.write().unwrap();
+        for x in snapshot_prior_roots {
+            writer.roots_original.insert(x);
+        }
+    }
+    */
```

Comment:
nit: Maybe this is just me, but I'd prefer if this were removed. Not a strong conviction though.

Reply:
I'm a fan of removing as well.  Issues can be used to track the details of future PRs better than littering in the code